app/core/settings.py

```diff
@@ -100,6 +101,10 @@
 
 STATIC_URL = '/static/'
 
+CONSUMER_GROUPS_HEADER = 'HTTP_X_CONSUMER_GROUPS'
+KONG_AUTH_HEADER = 'KONG-Authenticated'
+KONG_MODIFICATION_HEADER = 'KONG-Can-Modify'
```

Comment:
no need to tie it to Kong more than it already is, these two headers can be called `Digests-Can-Preview` and `Digests-Can-Modify`

Reply:
(as in, the middleware knows everything about Kong, but the rest of the application shouldn't)